app/src/sharedTest/java/org/oppia/app/topic/TopicFragmentTest.kt

```diff
@@ -313,7 +314,17 @@ class TopicFragmentTest {
           Matchers.containsString(topicName)
         )
       )
-    )
+    ).perform(scrollTo())
+  }
+  @Test
+  @UiThread
+  fun testTopicActivity_clickOnSeeMore_isPlayTabIsSelectedAndContentMatched() {
```

Comment:
I think this can be broken down into two separate tests

1. `testTopicActivity_clickOnSeeMore_playTabIsSelected`
2. `testTopicActivity_clickOnSeeMore_playContentIsDisplayed`

Reply:
`Optional Nit`: Also, you are using two different words with same meaning `Content` and `Data`, if possible use one of these words everywhere if possible.